lib/pavilion/schedulers/vars.py

```diff
@@ -155,7 +155,7 @@ def partition(self):
 
         return self._sched_config['partition'] or ''
 
-    def _test_cmd(self):
+    def _launch(self):
```

Comment:
You're still missing the alias.

Reply:
Wrap the alias as a `@dfr_var_method` - that way it will work regardless of whether or not the function is deferred in any child class.